### PR TITLE
Update dependencies, shutdown test server properly after running tests

### LINKED ISSUE #84: stop server after tests
It would be nice to stop the server after running the tests. 
From mocha version 4 the tests won't finish if the server is not shut down after running all tests.

https://mochajs.org/#--exit----no-exit

https://github.com/wikimedia/service-template-node/blob/43ead9407e81098242b89e44f195785d33000e52/package.json#L48

### PR DESCRIPTION
**Pull request by Pchelolo**
Fixes #84 

cc @wikimedia/services

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,26 +29,27 @@
...
-    "js-yaml": "^3.8.2",
+    "compression": "^1.7.1",
+    "domino": "^1.0.30",
+    "express": "^4.16.2",
+    "js-yaml": "^3.10.0",
     "preq": "^0.5.2",
-    "service-runner": "^2.2.5",
-    "swagger-router": "^0.5.6",
-    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master"
+    "service-runner": "^2.4.0",
```
Comment (d00rman) [inline package.json]: Let's bump it to 2.4.2. Also, let's bump preq to 0.5.3 once wikimedia/preq#30 is in.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.5.2",
+  "version": "0.6.0",
```
Comment (d00rman) [inline package.json]: I would prefer to still keep it on the 0.5.x branch, and switch to 0.6.0 once we start the switch to Node 8 in production.

**Review by d00rman — CHANGES_REQUESTED** [review]
LGTM, some minor comments in package.json, otherwise :+1:

**Comment by d00rman** [comment]
Oh, also, let's add "8" and "node" to .travis.yml please :)

**Comment by Pchelolo** [comment]
Fixed all @d00rman

**Review by d00rman — APPROVED** [review]

**Comment by d00rman** [comment]
Published as [v0.5.3](https://github.com/wikimedia/service-template-node/tree/v0.5.3)

**Pull request merged**